lib/GhcideSteal.hs

```diff
@@ -186,16 +186,18 @@ nameToLocation hiedb wsroot name = runMaybeT $
           erow <- liftIO $ findDef hiedb (nameOccName name) (Just $ moduleName mod) Nothing
           case erow of
             [] -> MaybeT $ pure Nothing
-            xs -> pure $ mapMaybe (defRowToLocation wsroot) xs
-        xs -> pure $ mapMaybe (defRowToLocation wsroot) xs
+            xs -> lift $ mapMaybeM (runMaybeT . (defRowToLocation wsroot imports)) xs
+        xs -> lift $ mapMaybeM (runMaybeT . (defRowToLocation wsroot imports)) xs
 
-defRowToLocation :: FilePath -> Res DefRow -> Maybe Location
-defRowToLocation wsroot (row :. info) =
+defRowToLocation :: Monad m => FilePath ->  M.Map ModuleName Uri -> Res DefRow -> MaybeT m Location
+defRowToLocation wsroot imports (row :. info) = do
```

Comment:
Does `MaybeT` really get us much here?

Reply:
whoops sorry didnt realize we could remove it entirely! (this was partially copied from the original fork)

Reply:
removed, thanks!